src/app/screens/Onboard/NewWallet/index.tsx

```diff
@@ -126,7 +126,7 @@ export default function NewWallet() {
                 </div>
                 <div className="float-right">
                   <QRCode
-                    value={`lndhub://${lndHubData.login}:${lndHubData.password}@${lndHubData.url}/`}
+                    value={`lndhub://${lndHubData.login}:${lndHubData.password}@${lndHubData.url}/`.toUpperCase()}
```

Comment:
actually. sorry... I am stupid and did not think when I thought about this QR code. 
We must noch change that one as it contains login data which would be altered and would no longer work. 
so we should leave it as it was.

Reply:
no worries did the required changes! kindly review and merge